packages/libs/react-ui/src/components/Table/Table.css.ts

```diff
@@ -1,81 +1,62 @@
-import { style } from '@vanilla-extract/css';
-import { atoms, token } from '../../styles';
+import { globalStyle, style, styleVariants } from '@vanilla-extract/css';
+import { token, uiSmallBold, uiSmallRegular } from '../../styles';
 
-export const tableWrapper = style([
-  atoms({
-    width: '100%',
-    overflowX: 'auto',
-  }),
-]);
+export const tableWrapper = style({
+  width: 'fit-content',
+  overflowX: 'auto',
+});
 
 export const table = style([
-  atoms({
-    borderRadius: 'sm',
-    border: 'hairline',
-    color: 'text.base.default',
-  }),
   {
+    color: token('color.text.base.default'),
+    borderCollapse: 'collapse',
     borderSpacing: 0,
-    borderColor: token('color.background.base.default'),
   },
 ]);
 
 export const headerRow = style([
-  atoms({
-    backgroundColor: 'base.default',
-  }),
+  {
```

Comment:
nestedheadres seem to have a strange borderradius in between.

![image](https://github.com/kadena-community/kadena.js/assets/4015521/070d5932-efff-4da9-a018-cf08888e9551)